### ISSUE TITLE
Collada animation

### ISSUE THREAD

**Issue by blksmth** [mrdoob/three.js#7244]
Hello everyone,

I am fairly new to webGL so please bear with me.

Basically, I outputted a simple collada animation via Blender to test — I load the dae file fine with the ColladaLoader, it shows up in my test, however the animation isn't playing. As far as I understand collada is the file type to use for animation, but why isn't my animation playing?

Thanks for an awesome build and easy to use webGL api.

Cheers

**Comment by eriadel** [comment]
Hello blksmth! I wanted just to post an issue regarding the DAE (collada) format animation.

Three JS has problems with importing correctly DAE filse (in our case from the DAZ studio) - not sure which Software you use. We have the same problem. Object loads, but no animation is played.

We assume DAZ exporter is working OK, as we can use exported DAE in Vue and Unity.

The same code works with the DAE objects from the 3js examples, so I guess not all DAE files are handled properly by the 3JS. For us it's clear - no animations - no serious game using this engine.

Best Regards,
www.artificialworlds.eu

**Comment by mrdoob** [comment]
We haven't implemented automatic animation playing in `ColladaLoader` yet. I'm planning on re-doing the `ColladaLoader` so I'll make sure we implement that next time.

**Comment by eriadel** [comment]
Hi mrdoob, could you enlight us what time approximatelly do you mean by the "next time" ? Asking, as we would be interested to use the threejs when this is fixed.

Best Regards,
www.artificialworlds.eu

**Comment by mrdoob** [comment]
Unfortunately I can't give you any dates...

**Comment by xirvr** [comment]
I've been wrangling with this for most of the day.  So far the only animation I can get working is from the loader example (the monster originating from: http://threejs.org/examples/webgl_loader_collada).  I've tried building a simple rotation with Cinema 4D, Maya, and Blender, in all cases exporting to DAE loads in place of the demo monster guy, but none of the traversed child.geometry objects have an animation property.

In looking at the DAE's themselves, it looks like the DAE's I'm exporting have some extra <animation> nesting between <library_animation> and <source> which the demo DAE does not.  I got as far as https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/ColladaLoader.js#L134 before figuring I should check for open issues.

If I'm on the right path here and it's not a ID10T error (which I'm not ruling out by any means), I'd be happy to help with this -- I have some anxious 3D animators waiting for me to get this figured out :)
